template.hbs

```diff
@@ -316,132 +383,184 @@
-          {{#each this}}
-          <td class="caption">{{@key}}</td>
-          {{/each}}
+      {{!-- XTest-> Measurement(s) (C11, C12, C13, C31, C32, C41, C43, C44) --}}
+      {{#typeOf this 'object'}}
+      <table class="main-table" style="margin-top: 0px;">
+        <tr class="table-rows">
+          <th class="table-headers">{{@key}} {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}
+          </th>
+          <td class="table-cells">
+            {{this.Value}} {{this.Unit}}
+          </td>
+          <td class="table-cells">
+            {{#if this.Minimum}}
+            min {{this.Minimum}} {{this.Unit}}
+            {{/if}}
+            {{#if this.Maximum}}
+            max {{this.Maximum}} {{this.Unit}}
+            {{/if}}
+          </td>
         </tr>
-        <tr>
-          <td class="caption"></td>
+      </table>
+      {{!-- XTest-> Measurement(s) (C10, C30, C40) --}}
+      {{else}}
+      <table class="main-table" style="margin-top: 0px;">
+        <tr class="table-rows">
+          <th class="table-headers">{{@key}}
+            {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}</th>
           {{#each this}}
-          <td>{{this}}</td>
+          <td class="table-cells" colspan="2">{{this}}</td>
           {{/each}}
         </tr>
       </table>
+      {{/typeOf}}
       {{/ifEqual}}
       {{/ifEqual}}
       {{!-- XTest-> C10, C30, C40, C41 --}}
       {{else}}
-      <table style="margin-top: -1px;">
-        <tr>
-          <th>{{@key}} {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}</th>
-          <td>{{this}}<em class="caption"></em></td>
+      <table class="main-table" style="margin-top: 0px;">
+        <tr class="table-rows">
+          <th class="table-headers">{{@key}}
+            {{ i18n @key "certificateFields" ../../Certificate.CertificateLanguages }}</th>
+          <td class="table-cells" colspan="2">{{this}}</td>
         </tr>
       </table>
       {{/typeOf}}
       {{/each}}
       {{/in}}
     </section>
-
     {{!-- Inspection-> C00, C01, C02, C03 --}}
     {{else}}
     {{#if this}}
-    <table>
-      <tr>
-        <th>{{@key}} {{ i18n @key "certificateFields" ../Certificate.CertificateLanguages }}</th>
-        <td>{{this}}</td>
+    <table class="main-table">
+      <tr class="table-rows">
+        <th class="table-headers">{{@key}} {{ i18n @key "certificateFields" ../Certificate.CertificateLanguages }}
+        </th>
+        <td class="table-cells">{{this}}</td>
+      </tr>
+    </table>
+    {{/if}}
+    {{/typeOf}}
+    {{/each}}
+  </section>
+
+  {{!-- OTHER TESTS --}}
+
+  <section>
+    <h2 class="main-title">{{ i18n "OtherTests" "certificateGroups" Certificate.CertificateLanguages}}</h2>
+    {{!-- <table class="main-table"> --}}
+    {{#each Certificate.OtherTests}}
+    {{!-- OtherTests-> SupplementaryInformation, NonDestructiveTests --}}
+    {{#typeOf this 'object'}}
```

Comment:
Could you replace SupplementaryInformation by OtherProductTests in the comment ?